interact.py

```diff
@@ -60,14 +61,13 @@ def sample_sequence(personality, history, tokenizer, model, args, current_output
         current_output = []
 
     for i in range(args.max_length):
-        instance, sequence = build_input_from_segments(personality, history, current_output, tokenizer, with_eos=False)
+        instance, _ = build_input_from_segments(personality, history, current_output, tokenizer, with_eos=False)
 
         input_ids = torch.tensor(instance["input_ids"], device=args.device).unsqueeze(0)
         token_type_ids = torch.tensor(instance["token_type_ids"], device=args.device).unsqueeze(0)
 
         logits = model(input_ids, token_type_ids=token_type_ids)
-
-        if "gpt2" == args.model:
+        if isinstance(logits, tuple):  # for gpt2 and maybe others
```

Comment:
@sshleifer what are the other values for `args.model` that work here apart from `openai-gpt` and `gpt2`? Are you planning on making changes to `train.py` and `interact.py` to support other models as well?

Reply:
only those work at the moment, feel free to send a PR adding more support!